modules/mediaforceBidAdapter.js

```diff
@@ -114,18 +114,28 @@ export const spec = {
     }
 
     const referer = bidderRequest && bidderRequest.refererInfo ? encodeURIComponent(bidderRequest.refererInfo.referer) : '';
+    const auctionId = bidderRequest && bidderRequest.auctionId;
+    const timeout = bidderRequest && bidderRequest.timeout;
     const dnt = utils.getDNT() ? 1 : 0;
-    let requests = [];
+    const requestsMap = {};
+    const requests = [];
+    let isTest = false;
     validBidRequests.forEach(bid => {
+      isTest = isTest || bid.params.is_test;
       let tagid = bid.params.placement_id;
       let bidfloor = bid.params.bidfloor ? parseFloat(bid.params.bidfloor) : 0;
-      let imp = [];
       let validImp = false;
       let impObj = {
         id: bid.bidId,
         tagid: tagid,
         secure: 1,
```

Comment:
Not new code but do you want to be checking `location.protocol` instead of hardcoding this?

Reply:
Fixed